class/Common/Breadcrumb.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace XoopsModules\Chess\Common;
+
+/*
+ You may not change or alter any portion of this comment or credits
+ of supporting developers from this source code or any supporting source code
+ which is considered copyrighted (c) material of the original comment or credit authors.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+/**
+ * Breadcrumb Class
+ *
+ * @copyright   XOOPS Project (https://xoops.org)
+ * @license     http://www.fsf.org/copyleft/gpl.html GNU public license
+ * @author      lucio <lucio.rota@gmail.com>
+ * @package     Chess
+ *
+ * Example:
+ * $breadcrumb = new Common\Breadcrumb();
+ * $breadcrumb->addLink( 'bread 1', 'index1.php' );
+ * $breadcrumb->addLink( 'bread 2', '' );
+ * $breadcrumb->addLink( 'bread 3', 'index3.php' );
+ * echo $breadcrumb->render();
+ */
+
+use XoopsModules\Chess;
+use XoopsModules\Chess\Common;
+
+\defined('XOOPS_ROOT_PATH') || exit('XOOPS Root Path not defined');
```

Comment:
should be commented out (or removed) so we're not mixing class/method definition with executable code (PSR).
```suggestion
//\defined('XOOPS_ROOT_PATH') || exit('XOOPS Root Path not defined');
```